workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/CommandUtil.java

```diff
@@ -75,5 +85,56 @@ public static boolean isInputValid(Object object) {
         
         return true;
 	}
+	
+	/**
+     * TODO remove this once Picocli supports case sensitive options
+     * @see https://github.com/remkop/picocli/issues/154
+     * @param args
+     * @return
+     */
+    public static String[] updateArgs(String[] args) {
+        if (args == null) {
+            return args;
+        }
+        for (int i = 0 ; i < args.length ; i++) {
+            if (args[i].equals(PROFILE_DIR_OPTION)) {
+                args[i] = TEMP_PROFILE_DIR_OPTION;
+            }
```

Comment:
Why are all these methods in CommandUtil.
They have a single responsibility of Manipulating internal args temporarily as a  workaround.
Use a different class like ProfileArgManipulator ? like this .
Write some JavaDoc on top on this class, the reason behind the manipulation